examples/pnpm-lock.yaml

```diff
@@ -9,21 +9,23 @@ specifiers:
   '@babel/parser': 7.18.9
   '@babel/preset-typescript': 7.18.6
   '@babel/types': 7.18.6
+  '@tsconfig/node16-strictest-esm': ^1.0.3
   '@types/jasmine': '*'
   '@types/node': 18.6.1
   date-fns: 2.29.1
   rxjs: '7'
   typescript: 4.8.2
 
 dependencies:
-  '@angular/compiler': 14.2.8_@angular+core@14.2.8
```

Comment:
This is still changing far more then necessary. If you revert and just run `pnpm install @tsconfig/node16-strictest-esm` does it still have this large diff? I'd think it should be 1 line...

Reply:
already fixed.